modules/appeals_api/app/swagger/legacy_appeals/v0/swagger_dev.json

```diff
@@ -111,9 +95,10 @@
                 "example": {
                   "errors": [
                     {
-                      "status": "404",
-                      "code": "veteran_not_found",
-                      "title": "Veteran Not Found"
+                      "title": "Resource not found",
```

Comment:
This difference in the 404 message comes from the `caseflow_request.rb` concern, which throws this error when the SSN can't be found in MPI